less/trickle-button.less

```diff
@@ -33,8 +33,15 @@
   // --------------------------------------------------
   &.is-full-width &__inner {
     position: fixed;
-    bottom: 0;
     left: 0;
     right: 0;
+
+    html:not(.is-nav-bottom) & {
```

Comment:
Code seems to suggest only top / bottom nav available (we could expand to left / right in the future). Would suggest each navigation position gets its own class and padding is added depending on which nav is active rather than not active.

html.is-nav-top &
html.is-nav-left &

etc